src/Umbraco.Tests/Composing/TypeFinderTests.cs

```diff
@@ -356,7 +356,6 @@ internal static IEnumerable<Assembly> GetBinAssemblies()
                     "Dynamic,",
                     "HtmlDiff,",
                     "Iesi.Collections,",
-                    "log4net,",
```

Comment:
serilog should be added here, you'd want to put `Serilog.` with the dot as i think that means anything that starts with serilog will be ignored

Reply:
Done